website/content/docs/architecture/scale.mdx

```diff
@@ -0,0 +1,283 @@
+---
+layout: docs
+page_title: Recommendations for operating Consul at scale
+description: >-  
+  When using Consul for large scale deployments, you can ensure network resilience by tailoring your network to your needs. Learn more about HashiCorp's recommendations for deploying Consul at scale.
+---
+
+# Operating Consul at Scale
+
+This page describes how Consul's architecture and impact its performance with large scale deployments, and shares recommendations for operating Consul in production at scale.
+
+## Overview
+
+Consul is a distributed service networking system deployed as a centralized set of servers that coordinate network activity using sidecars that are located alongside user workloads. When Consul is used for its service mesh capabilities, servers also generate configurations for Envoy proxies that run alongside service instances. These proxies support service mesh capabilities like end-to-end TLS and progressive deployments.
+
+Consul can be deployed in either a single datacenter or across multiple datacenters by establishing WAN federation or peering connections. In this context, a datacenter refers to a named environment whose hosts can communicate with low networking latency. Typically, users map a Consul datacenter to a cloud provider region such as AWS us-east-1 or Azure East US.
+
+To ensure consistency and high availability, Consul servers share data using the [Raft consensus protocol](/consul/docs/architecture/consensus). When persisting data, Consul uses Boltdb to store Raft logs and a custom file format for state snapshots. For more information, refer to [Consul architecture](/consul/docs/architecture).
+
+## General deployment recommendations
+
+This section provides general configuration and monitoring recommendations for operating Consul at scale.
+
+### Data plane resiliency
+
+To make service-to-service communication resilient against outages and failures, we recommend spreading multiple service instances for a service across fault domains:
+
+- Spread service instances across _multiple infrastructure-level availability zones_.
+- Spread service instances across _multiple runtime platform instances_, such as multiple Kubernetes clusters.
+- Spread service instances across _multiple Consul datacenters_.
```

Comment:
could we condense/remove "spread service instances across"? it seems repetitive, esp when we introduce this with "we recommend spreading multiple service instances..."